Makefile

```diff
@@ -20,6 +20,9 @@ osbs-build:
 	# hack/build.sh
 	./tmp/build/build.sh
 
+unit:
+	go test -v ./pkg/...
```

Comment:
Is this what other teams are doing? We should ensure that this doesn't cause trouble when launched by the CI framework. Is there a way to do that?

Reply:
It's being used exactly this way by the OLM team. I actually just copied their pattern for this bit:

https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Makefile#L21

https://github.com/openshift/release/blob/master/ci-operator/config/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master.yaml#L38